runtime/doc/vim_diff.txt

```diff
@@ -85,6 +85,10 @@ of these in your config by simply removing the mapping, e.g. ":unmap Y".
 	nnoremap <C-L> <Cmd>nohlsearch<Bar>diffupdate<CR><C-L>
 	inoremap <C-U> <C-G>u<C-U>
 	inoremap <C-W> <C-G>u<C-W>
+	nmap ]g <Plug>DiagnosticGotoNext
```

Comment:
It's considered "best practice" to wrap `<Plug>` mappings with parenths, like: `<Plug>(DiagnosticGotoNext)`. I think it may also make sense to say `<Plug>(VimDiagnosticNext)`. Prefixing with `vim` makes sense to me because it's `vim.diagnostic.goto_next()`.

Just a few thoughts.